Files/Controllers/TerminalController.cs

```diff
@@ -70,8 +70,8 @@ private async Task LoadAsync()
 
         public async void Init()
         {
-            await LoadAsync();
-            await GetInstalledTerminalsAsync();
+            //await LoadAsync();
+            //await GetInstalledTerminalsAsync();
```

Comment:
Is this change required for this pull request?

Reply:
I commented it out because it wouldn't let me compile. I'll revert the comments once the PR is done